capa/helpers.py

```diff
@@ -57,6 +58,10 @@ def get_format_from_extension(sample: str) -> str:
         return FORMAT_SC32
     elif sample.endswith(EXTENSIONS_SHELLCODE_64):
         return FORMAT_SC64
+    elif sample.endswith(EXTENSIONS_CAPE):
+        # once we have support for more sandboxes that use json-formatted reports,
+        # we update this logic to ask the user to explicity specify the format
```

Comment:
good, and/or guess at the content by looking for expected content, like a CAPE version number or the keys typically used by the format.

Reply:
thanks for the suggestion! I think in the future we should probably have one extension (EXTENSIONS_TRACE) that contains all report file formats (json, etc), and then if a sample has that extension we'd try to guess the format (FORMAT_CAPE, FORMAT_VMRAY, etc.) based on indicators in the trace file.